src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1418,6 +1418,122 @@ def aks_upgrade(cmd, client, resource_group_name, name, kubernetes_version, no_w
     return sdk_no_wait(no_wait, client.create_or_update, resource_group_name, name, instance)
 
 
+def aks_use_devconnect(cmd, client, cluster_name, resource_group_name, space_name='default'):  # pylint: disable=line-too-long
+    """
+    Use Azure Dev Connect with a managed Kubernetes cluster.
+
+    :param cluster_name: Name of the target AKS cluster.
+    :type cluster_name: String
+    :param resource_group_name: Name of the target AKS cluster's resource group.
+    :type resource_group_name: String
+    :param space_name: The isolated space in the cluster to develop in.
+    :type space_name: String
+    """
+
+    vsce_tool = 'Visual Studio Connected Development Services'
+    vsce_cli = 'vsce'
+    install_vsce = False
+    system = platform.system()
```

Comment:
Need to add a final `else` statement for if the value of `system` does not equal `Linux, Windows or Darwin`
Other values are possible: https://docs.python.org/3/library/platform.html#platform.system